src/models/treatment_models.py

```diff
@@ -4,6 +4,8 @@
 
 import numpy as np
 
+from src.utils import get_timeseries
+
 
 class TreatmentModel(object):
```

Comment:
This comment refers to line 35 below: I wonder if the insulin model justification notebook should live in this repository under notebooks?

Reply:
Yep, that seems like a good idea